bigframes/geopandas/geoseries.py

```diff
@@ -92,6 +92,15 @@ def to_wkt(self: GeoSeries) -> bigframes.series.Series:
         series.name = None
         return series
 
+    def buffer(self: GeoSeries, distance: float) -> bigframes.series.Series:  # type: ignore
+        return self._apply_unary_op(ops.GeoStBufferOp(distance=distance))
```

Comment:
Should raise `NotImplementedError` because in BigQuery, `distance` is in meters.